src/services/completions.ts

```diff
@@ -815,6 +815,12 @@ namespace ts.Completions {
         const isJavaScriptFile = isSourceFileJavaScript(sourceFile);
 
         let isJsDocTagName = false;
+        // This is for the case when users request completion in JsDoc without "@"
+        // i.e.
+        //      /**
+        //       * |completion here|
+        //       **/
+        let shouldAppendAtSignBeforeJsDocTagName = false;

```

Comment:
nit, consider switching this to `isJSDocTag`, where `@param` is a tag, and `param` is a tag name.